### PR TITLE
[66] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by cherishebojudah**
## Task: `[66] Style the Contact Us Page using CSS
Contact page web
Contact page tablet 1
Contact page tablet 2
Contact page mobile 1
Contact page mobile 2

`
- `Styled the e-commerce contact page using css`

### CODE REVIEW

File: submissions/ebojudahCherish/css/styles.css [hunk 1]
```diff
@@ -567,18 +572,26 @@ img {
 .register-fieldset, .contact-form fieldset {
     width: 350px;
     justify-content: center;
-    margin: auto;
+    margin: a
```
Comment (Ireti203) [inline submissions/ebojudahCherish/css/styles.css:575]: What are you styling as "margin : a" ? Kindly remove if it is not being used.

**Review by adaugochi — CHANGES_REQUESTED** [review]
I can't see your styling for the contact form page. The only CSS file on your PR is that which you created before (i.e for signup page). Or am I missing something?

**Comment by cherishebojudah** [comment]
> I can't see your styling for the contact form page. The only CSS file on your PR is that which you created before (i.e for signup page). Or am I missing something?

All the pages share the same style sheet because they share a lot of properties, in sections where they don't, I indicate with a comment in the stylesheet.

**Comment by cherishebojudah** [comment]
And also, I've mostly done the styling for all pages, just some minor changes I left out since I'm creating different PRs for each,  so I just update the stylesheet when I need to make a new PR for a different page.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**